test/jdk/java/lang/invoke/common/test/java/lang/invoke/lib/InstructionHelper.java

```diff
@@ -148,4 +159,199 @@ public static String csym(Class<?> c) {
     public static String cref(Class<?> c) {
         return methodType(c).toMethodDescriptorString().substring(2);
     }
+
+
+    // loadCode(MethodHandles.Lookup, String, MethodType, Consumer<? super MethodHandleCodeBuilder<?>>) et al...
+
+    public static MethodHandle loadCode(MethodHandles.Lookup lookup, String name, MethodType type, Consumer<? super MethodHandleCodeBuilder<?>> builder) {
+        String className = generateClassNameFromLookupClass(lookup);
+        return loadCode(lookup, className, name, type, builder);
+    }
+
+    public static MethodHandle loadCode(MethodHandles.Lookup lookup, String className, String methodName, MethodType type, Consumer<? super MethodHandleCodeBuilder<?>> builder) {
```

Comment:
Are these two `loadCode` methods taking a builder `Consumer` used by the tests?

If so, the following classes and methods can be made private to hide them as internals.

Reply:
Comments applied